embassy-stm32/src/rcc/f1.rs

```diff
@@ -24,10 +24,15 @@ pub struct Config {
     pub pclk1: Option<Hertz>,
     pub pclk2: Option<Hertz>,
     pub adcclk: Option<Hertz>,
+    pub pllxtpre: Option<bool>,
```

Comment:
I think this should be a `bool`, not an `Option<bool>`? `true/false` mean use it / don't use it, `None` doesn't make much sense.